APIKit/APIKit.swift

```diff
@@ -146,13 +111,17 @@ public class API: NSObject, NSURLSessionDelegate, NSURLSessionDataDelegate {
         }
     }
 
-    // send request and build response object
+    // this methods can be removed in Swift 1.2
```

Comment:
Why this is needed for Swift 1.1? If this is really required, it could be better to omit the default parameter value from the overloaded method, I think.